keyboards/sendyyeah/pix/readme.md

```diff
@@ -0,0 +1,15 @@
+# Pix
+
+![Pix]()
```

Comment:
Need an image here. Upload it to Imgur and add `l` to the end of the filename to get a large thumbnail-sized image: https://api.imgur.com/models/image#thumbs

Reply:
sure, will do

Reply:
done